docs/services/rumqttd.md

```diff
@@ -0,0 +1,37 @@
+# rumqttd
+
+A high performance, embeddable [MQTT](https://en.wikipedia.org/wiki/MQTT) broker installed via [mother-of-all-self-hosting/ansible-role-rumqttd](https://github.com/mother-of-all-self-hosting/ansible-role-rumqttd).
```

Comment:
It's important to link to the project website on Github (or elsewhere, if available).